nengo_viz/static/viz_scale.js

```diff
@@ -23,8 +18,8 @@ VIZ.scale.zoom = function (wheel, event){
 		var offsetY = VIZ.pan.cposn.ul.y;
 	}
 	else{
-		var offsetX = event.offsetX;
-		var offsetY = event.offsetY;
+		var offsetX = event.offsetX || event.clientX;
+		var offsetY = event.offsetY || event.clientY;
```

Comment:
Hmm...  not sure what this is doing.  Why is this code looking in two different places for the x,y values?  In what situations are they in one place or in the other?

Also, note that if ```event.offsetX``` is 0 and ```event.clientX``` is undefined, then this will set ```offsetX``` to ```undefined``` (since ```0 || undefined``` evaluates to ```undefined```).  Is this the intended behaviour?